administration/src/Router.tsx

```diff
@@ -100,14 +100,7 @@ const Router = () => {
       },
     ]
     return createBrowserRouter(routes.filter((element): element is RouteObject => element !== null))
-  }, [
-    authData,
-    projectConfig.applicationFeature,
-    signIn,
-    signOut,
-    projectConfig.cardStatistics,
-    projectConfig.cardCreation,
-  ])
+  }, [authData, signIn, signOut, projectConfig])
```

Comment:
is this really working meanwhile with stringify or anything to use objects as deps?

Reply:
This should work (and is as far as I can see). But it depends. If the object is recreated every render, e.g. as in [this example](https://react.dev/reference/react/useEffect#removing-unnecessary-object-dependencies) of the react docs, the hook is executed again on every render.
However, if the object is not redefined/does not change on every render, this works just fine. This is e.g. the case if it is a value from `useState`, `useMemo`, `useContext` or defined outside of a react component, which is the case here.